laika/reports.py

```diff
@@ -1145,6 +1146,22 @@ def _drive_call(self, method, *args, **kwargs):
                 else:
                     raise
 
+    def _query_arguments(self, base_arguments):
+        """
+        Adds the required shared drive parameters to arguments in case
+        drive_id is specified
+        """
+        if self.drive_id is not None:
+            args = base_arguments.copy()
+            if args.get('parents', None):
+                args['parents'][0].update({'kind': 'drive#fileLink',
```

Comment:
May a file have more than one parent? If it does, and apparently it's the case since otherwise PyDrive or GoogleDriveAPI wouldn't have a metadata field called `parents` which is a list), I would suggest a more general implementation; something like:
```
for parent in args.get('parents', []):
    parent.update({'kind': 'drive#fileLink', 'driveId': self.drive_id})
```